pandas/tests/io/test_common.py

```diff
@@ -466,36 +472,38 @@ def test_unknown_engine(self):
             with pytest.raises(ValueError, match="Unknown engine"):
                 pd.read_csv(path, engine="pyt")
 
-    def test_binary_mode(self):
-        """
-        'encoding' shouldn't be passed to 'open' in binary mode.
 
-        GH 35058
-        """
-        with tm.ensure_clean() as path:
-            df = tm.makeDataFrame()
-            df.to_csv(path, mode="w+b")
-            tm.assert_frame_equal(df, pd.read_csv(path, index_col=0))
+def test_binary_mode():
```

Comment:
this test and `test_warning_missing_utf_bom` had nothing to do with mmap but were in its test class.